collections/concepts/access-controls.md

```diff
@@ -12,16 +12,266 @@ width: 1200
+perform analytics, and a third system is only allowed to [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) the data 
+to an integrated 3rd party but never access the raw data.
+
+<img src="/assets/images/concepts/applications_each_system.jpeg">
+
+#### Enable partners to use data secured by Basis Theory
+
+Often, businesses need to grant their partners access to their sensitive data — 
+traditionally, these businesses would just export the needed data and send it to their partner. 
+With Basis Theory Applications, a business can create an Application that is restricted to ONLY the data that partner needs.
+In some cases, businesses will not allow their partners direct access to the raw data,
+but rather only allow them to [Proxy](https://developers.basistheory.com/concepts/what-is-the-proxy/) the data to 
+another API or execute serverless workloads on the sensitive data using Reactors.
+
+<img src="/assets/images/concepts/applications_partners.jpeg">
+
+#### Manage Basis Theory Assets with Infrastructure as Code
+
+The Management application type allows the Basis Theory platform to be configured entirely via our API. 
+This enables engineers to write scripts or to use IaC (Infrastructure as  Code) tooling such as 
+[Terraform](/guides/manage-applications-with-terraform/) to manage your Tenants and Services without logging into our Portal UI.
+
+<img src="/assets/images/concepts/applications_iac.jpeg">
+
+
+## What are Access Rules?
+
+Access Rules are the building blocks for constructing fine-grained access control policies for an Application.
+When performing an operation through the Basis Theory API, Access Rules are evaluated in priority order until 
+the first rule is found that matches the operation and target resource(s). Once a matching rule is identified, 
+the [transform](/concepts/access-controls#what-are-access-rules-transform) defined on the rule determines
+if and how Token data will be returned from the API. If no matching rules are found, 
+access to the requested resource is denied with a `403 Forbidden` error.
 
 <span class="base-alert info">
-    <span>
-      An improved access control system is coming soon! In the meantime, check out some of our related docs.
-    </span>
+  <span>
+    Access Rules are currently only supported on Private and Public Application types, and only control access to 
+    Token resources. Access Rules are not supported on Management Applications at this time. 
+  </span>
 </span>
 
-- [What are Applications?](/concepts/what-are-applications)
-- [Understanding Permissions](/concepts/understanding-permissions)
-- [Token Containers](/concepts/what-are-token-containers)
+Access Rules are comprised of the following properties:
+
+### Description
+
+A short description of the rule to help you identify its purpose.
+
+### Container
+
+Access Rules can be scoped to a specific set of resources using Containers. 
```

Comment:
i wouldn't say resources here, as it might be confusing, i would prob just say tokens